ConeSearch.tex

```diff
@@ -154,13 +154,25 @@ \section{Service Interface Requirements}
 				\item[\textbf{DEC}] a declination
 					in the ICRS coordinate system for the positon of the center of the cone
 					to search, given in decimal degrees.
-				\item[\textbf{SR}] the radius of
-					the cone to search, given in decimal degrees.
+				\item[\textbf{SR}] the radius of the cone to search, given in decimal degrees.
+					If set to zero (SR=0) it should not return an error condition because the
+					request might describe an interest in retrieving only the metadata structure
+					of the service's response (i.e. discovery of the FIELDS delivered by the service).
+					This is similar to setting MAXREC=0, i.e. a service metadata request as prescribed
+					by DALI\footnote{SR=0 is kept in this version of this specification for back
+					compatibility. It is suggested to prefer the usage of MAXREC to enable metadata
+					discovery}\citep{2017ivoa.spec.0517D}.
 			\end{description}
 			\begin{bigdescription}
 				\item[Example]
 					\url{http://mycone.org/cgi-bin/search?RA=180.567&DEC=-30.45&SR=0.0125}
 			\end{bigdescription}
+		\item As defined by DALI a SHOULD also understand the following parameter:
```

Comment:
I think a word is missing here. I guess "a **service** SHOULD also" or something similar.

Reply:
Commit 249d658 should have fied this.
Thanks for spotting the missing word!